app/views/api/v2/report_templates/main.json.rabl

```diff
@@ -1,5 +1,8 @@
 object @report_template
 
 extends "api/v2/report_templates/base"
+extends "api/v2/layouts/permissions"
 
-attributes :description, :created_at, :updated_at
+attributes :description, :snippet, :locked, :vendor, :created_at, :updated_at
+
+node(:available_actions) { |t| { generatable: !t.snippet?, lockable: !t.locked?, unlockable: t.locked? } }
```

Comment:
This I'll probably change to plain `attributes`.

Reply:
I think it would be better to have these as plain attributes.